redash/query_runner/oracle.py

```diff
@@ -137,6 +137,17 @@ def output_handler(cls, cursor, name, default_type, length, precision, scale):
                     arraysize=cursor.arraysize,
                 )
 
+    @classmethod
+    def calculate_line_and_column(query, offset):
```

Comment:
Any possibility of adding a test for this new function? :smile:

Reply:
I understand.

I will try to add the test.